.github/workflows/lint-rust.yml

```diff
@@ -38,8 +38,7 @@ jobs:
           save-if: ${{ github.ref_name == 'main' }}
 
       - name: Run cargo clippy with all features enabled
-        # not all features can combine with each other for nano-arrow
-        run: cargo clippy --workspace --all-targets --exclude nano-arrow --all-features -- -D warnings
+        run: cargo clippy -p polars --all-features -- -D warnings
```

Comment:
@stinodego I had to change this as it seem that cargo activated/toggle default features of all crates simultanously. Whereas the polars crate determines which features combine with eachother. They should trickle downwards.